api/external.go

```diff
@@ -109,22 +109,25 @@ func (a *API) internalExternalProviderCallback(w http.ResponseWriter, r *http.Re
 
 	providerType := getExternalProviderType(ctx)
 	var userData *provider.UserProvidedData
-	var providerToken string
+	var providerAccessToken string
+	var providerRefreshToken string
 	if providerType == "twitter" {
 		// future OAuth1.0 providers will use this method
 		oAuthResponseData, err := a.oAuth1Callback(ctx, r, providerType)
 		if err != nil {
 			return err
 		}
 		userData = oAuthResponseData.userData
-		providerToken = oAuthResponseData.token
+		providerAccessToken = oAuthResponseData.token
+		providerRefreshToken = oAuthResponseData.refreshToken
```

Comment:
oauth1 doesn't have the concept of a refresh token - this is only specific to oauth2 so let's remove it for oauth1 providers

Reply:
Oh right my bad, fixed in https://github.com/supabase/gotrue/pull/641/commits/56e429dce64e8021b5e629e6a4b98962cdc4aabf let me know if that's fine with you